src/Backend/OnboardingInsuranceAPI/Areas/Insurance/InsuranceDataRequest.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace OnboardingInsuranceAPI.Areas.Insurance;
 
-public record InsuranceData
+public class InsuranceDataRequest
```

Comment:
Why change it to class?

Reply:
Oh. I created new class and forgot change to record. Git count this like change.